workers/ohsome_quality_analyst/indicators/mapping_saturation/metrics.py

```diff
@@ -0,0 +1,29 @@
+"""Collection of common error metrics as functions."""
+
+import numpy as np
+from numpy import float64, ndarray
+
+
+def me(actual: ndarray, predicted: ndarray) -> float64:
+    """Mean Error"""
+    return np.mean(np.subtract(actual, predicted))
+
+
+def mse(actual: ndarray, predicted: ndarray) -> float64:
+    """Mean Squared Error"""
+    return np.mean(np.square(np.subtract(actual, predicted)))
+
+
+def rmse(actual: ndarray, predicted: ndarray) -> float64:
+    """Root Mean Squared Error"""
+    return np.sqrt(mse(actual, predicted))
+
+
+def nrmse(actual: ndarray, predicted: ndarray) -> float64:
+    """Normalized Root Mean Squared Error"""
+    return rmse(actual, predicted) / (actual.max() - actual.min())
+
+
+def mae(actual: ndarray, predicted: ndarray) -> float64:
+    """Mean Absolute Error"""
+    return np.mean(np.abs(np.subtract(actual, predicted)))
```

Comment:
Remove metrics file since only one metric is used. Implement metric directly in base model class.
 https://github.com/GIScience/ohsome-quality-analyst/pull/170/commits/e8e5df0891672c643733abba7b39d18a03913515